packages/server/server/api/destinations/destination.controller.ts

```diff
@@ -16,7 +16,8 @@ class DestinationController implements Controller {
 
     private initializeRoutes() {
         this.router.post(this.path, this.createDestination.bind(this));
-        this.router.get(this.path, this.getDestinationsByTrecipeId.bind(this));
+        this.router.get(this.path, this.getDestinationByPlaceId.bind(this));
+        this.router.get(`${this.path}/in`, this.getDestinationsByTrecipeId.bind(this));
```

Comment:
Definitely. Added passportAuth to getDestsByTrecipeId and also createDestination too, since I destination only gets created when user starts adding it to their trecipes.